api/src/main/resources/org/openmrs/api/db/hibernate/OrderSet.hbm.xml

```diff
@@ -36,6 +36,9 @@
             </type>
         </property>
 
+        <many-to-one name="concept" class="org.openmrs.Concept"
```

Comment:
Now you have got this right.

Reply:
OK!! So I have been looking through the error logs trying to figure out what's missing of which I know is as a result in the changeset. Could you help me out on this one as well

Reply:
oh wait a minute, i think you got the name wrong. Just take a looking at existing ones and their corresponding java classes to fix this yourself.

Reply:
By existing ones, I hope you mean existing changesets or hbms